templates/roothints.service.epp

```diff
@@ -0,0 +1,9 @@
+<%- | Stdlib::Absolutepath $hints_file, Stdlib::HTTPSUrl $root_hints_url| -%>
+# THIS FILE IS MANAGED BY PUPPET
+# BASED ON https://wiki.archlinux.org/title/Unbound#Roothints_systemd_timer
+[Unit]
+Description=Update root hints for unbound
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/curl --output <%= $hints_file %> <%= $root_hints_url %>
```

Comment:
I guess this should use `unbound::fetch_client` and not `curl`?

Reply:
I considered this, but! `unbound::fetch_client` doesn't have an absolute path and `ExecStart` requires this :( Should we use `ExecStart=/usr/bin/$unbound::fetch_client`? That's quite ugly, but it seems to be fine on CI.

Reply:
it's better than using `curl` here and `wget` there :)

also, I thought recent systemd is OK with relative paths here?

Reply:
okay I tested it locally and it works with a relative path. let's see what CI says.

Reply:
Seems EL7 is unhappy about that

Reply:
I hate old software so much